zoltar-core/src/main/java/com/spotify/zoltar/fs/FileSystemExtras.java

```diff
@@ -67,6 +67,9 @@ public static Path path(final URI uri) throws IOException {
     // uri with no scheme might be a local path
     if (scheme == null) {
       return Paths.get(uri.toString());
+    } else if (scheme.equals("gs")) {
+      // GCS requires that the path has a trailing slash. Resolve adds the slash only if needed.
+      return Paths.get(uri.resolve("/"));
```

Comment:
@regadas Here's the update to handle GCS. PTAL when you have a chance.

Reply:
@rculbertson hm, `resolve` will not play nice here. It only works well when you are trying to append (resolve) a relative path to a give base `URI`

```java
URI.create("gs://bucket/name").resolve("/")
java.net.URI = gs://bucket/
```